map_gui.cpp

```diff
@@ -1037,6 +1040,11 @@ void MapGui::render(Renderer& renderer) {
   auto currentTimeReal = clock->getRealMillis();
   lastHighlighted = getHighlightedInfo(size, currentTimeReal);
   renderMapObjects(renderer, size, currentTimeReal);
+  if(auto *rinst = fx::FXRenderer::getInstance()) {
+	float zoom = float(layout->getSquareSize().x) / 24.0f;
```

Comment:
Please get the 24 value from Renderer::getNominalSize()

Also there is a tab here and in a few other places.